apps/meteor/app/integrations/server/api/api.js

```diff
@@ -51,7 +52,7 @@ function buildSandbox(store = {}) {
 		},
 	};
 	Object.keys(Models)
-		.filter((k) => !k.startsWith('_'))
+		.filter((k) => !forbiddenModelMethods.includes(k))
```

Comment:
Won't this allow accessing __proto__ stuff and so on?

Reply:
Not really, this is the way the previous code was doing. I know, I agree that we should not expose all the models inside the integration script, but the old version was doing this. @rodrigok @ggazzo @sampaiodiego any thoughts on this?